### PR TITLE
Add Gradle 2.14.1

### PR DESCRIPTION
**Pull request by psg9999**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [ ] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ ] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [ ] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/master/README.md#acceptable-formulae)?

-----

Closes https://github.com/Homebrew/homebrew-versions/pull/1396

### CODE REVIEW

File: gradle2141.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
-class Gradle221 < Formula
-  desc "Build system based on the Groovy language"
+class Gradle2141 < Formula
```
Comment (MikeMcQuaid) [inline gradle2141.rb]: Sorry, just noticed: this should be `Gradle214` as we use major/minor not patch/security versions.

**Comment by poemdexter** [comment]
Hey @MikeMcQuaid can you approve?  :)

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `gradle214 2.14.1 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md). Thanks!

**Comment by psg9999** [comment]
Will do.  Thanks Mike.

**Pull request merged**